index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Evans_Portfolio</title>
+    <script
+      src="https://kit.fontawesome.com/0ec53d8e70.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header id="navigation">
+      <div class="logo">
+        <h3>Evans</h3>
+      </div>
+      <nav class="navlinks">
+        <ul class="links">
+          <li class="link"><a href="#portfolio">Portfolio</a></li>
+          <li class="link"><a href="#about">About</a></li>
+          <li class="link"><a href="#contact">Contact</a></li>
+          <li>
+            <a href="#"><i class="fa-regular fa-envelope"></i></a>
+          </li>
+        </ul>
+      </nav>
+      <div class="toggle">
+        <div class="menu"></div>
+      </div>
+    </header>
+
+    <section id="hero">
+      <div class="hero_text">
+        <h1 class="title">
+          Hey There <br />
+          I'm Evans
+        </h1>
+        <h2 class="subtitle">I'm a Software Developer</h2>
```

Comment:
- [ ] I noticed that the margin before and after the `I'm a Software Developer` is the same in the Figma design but not in our 🙂 own, kindly effect this to have a pixel-perfect design like you'll soon be doing in your full-time job.

- Figma design
<img width="217" alt="right" src="https://user-images.githubusercontent.com/41163571/227010023-9fcfb014-309c-4293-ba46-014f8e1c0000.png">

- Your design
<img width="169" alt="wrong" src="https://user-images.githubusercontent.com/41163571/227179391-9b13fdf2-2533-46d2-9297-7041dd5b8380.png">